package.json

```diff
@@ -20,25 +20,56 @@
     "url": "https://github.com/mjbvz/vscode-github-markdown-preview-style/issues"
   },
   "engines": {
-    "vscode": "^1.15.0"
+    "vscode": "^1.41.0"
   },
   "categories": [
     "Other"
   ],
+  "activationEvents": [
+    "onLanguage:markdown",
+    "onWebviewPanel:markdown.preview",
+    "onCustomEditor:vscode.markdown.preview.editor"
+  ],
+  "main": "dist/extension.js",
+  "browser": "dist/extension.js",
   "contributes": {
+    "markdown.markdownItPlugins": true,
     "markdown.previewStyles": [
-      "./base.css",
-      "./github-markdown.css",
-      "./node_modules/highlight.js/styles/github.css"
+      "./dist/base.css",
+      "./dist/github-markdown.css",
+      "./dist/highlight-js.css"
+    ],
+    "configuration": [
+      {
+        "title": "Markdown Preview Github Styling",
+        "properties": {
+          "markdown-preview-github-styles.colorMode": {
```

Comment:
[nit] Let's rename this to `Color Theme` instead of `color mode`. That's the term VS Code uses in a few places, such as `workbench.preferredLightColorTheme`

Reply:
Done as of b2c8012. Was thinking about this as I didn't want users to think there was arbitrary themes available à la GitHub.